src/index.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash';
+import printMe from './print.js';
+
+ function component() {
+   const element = document.createElement('div');
+  const btn = document.createElement('button');
+
+   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
+  btn.innerHTML = 'Click me and check the console!';
+  btn.onclick = printMe;
+
+  element.appendChild(btn);
+
+   return element;
+ }

```

Comment:
- [ ] According to the requirement you should create an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. As mention by first reviewer

Reply:
Ok.

Reply:
This is resolved. The screenshot is attached.

https://user-images.githubusercontent.com/106825373/194581023-292c3f49-4044-4a21-a383-daa4a316b3a6.png